lib/svtplay_dl/utils/check_ffmpeg.py

```diff
@@ -0,0 +1,63 @@
+import os, sys
+from subprocess import check_output
+from pkg_resources import parse_version
+from re import findall, sub, split
+from requests import get
+from svtplay_dl.utils import which
+from svtplay_dl import log
+
+def is_old():
+    if not which('ffmpeg'):
+        return None
+    version_check = check_output([which('ffmpeg'),'-version']).split('\n')[0].split(' ')[:3]
+    version_check = [v for v in version_check if findall('\d+', v)][0]
+    if 'N' in version_check[0]:
+        k = 1 if version_check.endswith('static') else 0
+        version_check = version_check.split('-')[::-1][k][1:]
+
+        r = get('https://api.github.com/repos/FFmpeg/FFmpeg/commits/' + version_check)
+        print ' '.join(split('T|Z', r.json()['commit']['committer']['date'])[:2])
```

Comment:
oops, thats not supposed to be there, sorry. has been a while since i touched this. :) 👍